index.html

```diff
@@ -0,0 +1,660 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>Youtube Page</title>
+  </head>
+  <body>
+    <nav class="flex">
+      <div id="left-navbar" class="flex">
+        <i class="fa fa-bars" aria-hidden="true"></i>
+        <img
+          src="assets/YouTube-Logo.svg"
+          style="width: 80px;"
+          height:80px;
+          alt=""
+        />
+      </div>
+      <div id="mid-navbar" class="flex">
+        <input id="search-bar" type="text" placeholder="Search" />
+        <button id="search-button">
+          <i class="fa fa-search" aria-hidden="true"></i>
+        </button>
+      </div>
+
+      <div id="right-navbar" class="flex">
+        <i class="fa fa-video-camera" aria-hidden="true"></i>
+        <i class="fa fa-th fa-lg" aria-hidden="true"></i>
+        <i class="fa fa-bell" aria-hidden="true"></i>
+        <i class="fa fa-envelope" aria-hidden="true"></i>
+      </div>
+    </nav>
+
+    <div class="d-flex space-between">
+      <div class="main-fc d-flex flex-column">
+        <div class="container">
+          <div class="main-video">
+            <iframe
+              width="853"
+              height="480"
+              src="https://www.youtube.com/embed/YbYWhdLO43Q"
+              frameborder="0"
+              allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
```

Comment:
Please remove all inline styling and use your CSS instead.

Reply:
Hi Zakariae! According to this article entitled "[HTML Style Guide and Coding Conventions,](https://www.w3schools.com/html/html5_syntax.asp)" it is recommended to always specify alt, width, and height for Images. As much as possible, we did not use inline styling, except for the case of images. We would appreciate your consideration on this matter.